s2i-mvn-gradle/Dockerfile

```diff
@@ -0,0 +1,60 @@
+# openjdk-gradle
+FROM registry.access.redhat.com/rhel 
+
+# TODO: Put the maintainer name in the image metadata
+LABEL maintainer="Abhishek Singh <abhishek@linux.com>" \
+      io.k8s.description="Builder Image for building Java applications with Maven 3.5.3 or Gradle 4.8 on RHEL 7" \
+      io.k8s.display-name="S2I builder 1.0" \
```

Comment:
Can the display name be more descriptive of the gradle builder?

Reply:
@sabre1041 One thing that was suggested to me when I had a Spring Boot+Gradle customer was to leverage the current openjdk-s2i base image (to keep full support; removal of bits can degrade support level) and use Maven to trigger the Gradle build.  Yeah, kinda ridiculous but I understood the point about support.